extra/modules/wurfl-devicedetection/src/main/java/org/prebid/server/hooks/modules/com/scientiamobile/wurfl/devicedetection/v1/ExtWURFLMapper.java

```diff
@@ -0,0 +1,73 @@
+package org.prebid.server.hooks.modules.com.scientiamobile.wurfl.devicedetection.v1;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import lombok.Builder;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+
+@Builder
+@Slf4j
+public class ExtWURFLMapper {
+
+    private final Set<String> staticCaps;
+    private final Set<String> virtualCaps;
+    private boolean addExtCaps;
+    private final com.scientiamobile.wurfl.core.Device wurflDevice;
+    private static final String NULL_VALUE_TOKEN = "$null$";
+    private static final String WURFL_ID_PROPERTY = "wurfl_id";
+
+    public JsonNode mapExtProperties() {
+
+        final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Create it once or use `ObjectMapperProvider.mapper()` (see if its configuration breaks your logic)